whitepaper/summary.tex

```diff
@@ -0,0 +1,46 @@
+\setcounter{chapter}{0}
+\chapter*{Summary}
+\def\chpname{summary}\label{chp:\chpname}
+\addcontentsline{toc}{section}{Summary}
+\markboth{}{}
+
+% Short (c. 250 word) summary for arxiv posting:
+
+\noindent
+The Large Synoptic Survey Telescope is designed to provide a single
```

Comment:
@ivezic I omitted to call this out for you to look at in the PR thread or latex comments, sorry - great if you can check and modify my wording for this, the "arxiv abstract". Thanks!